<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelescopeView</class>
 <widget class="QMainWindow" name="TelescopeView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="zoomIn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoomOut">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
      <item>
       <widget class="QScrollArea" name="imageScrollArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="widgetResizable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="scrollAreaImageLabelContent">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>778</width>
           <height>452</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="connectButton">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loadImageButton">
          <property name="text">
           <string>Load image from file</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_screenshot"/>
    <addaction name="actionClose_2"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionImage_processing_settings"/>
    <addaction name="separator"/>
    <addaction name="actionSettingsScrews"/>
    <addaction name="separator"/>
    <addaction name="actionConnection"/>
    <addaction name="separator"/>
    <addaction name="actionDebug_Image"/>
   </widget>
   <widget class="QMenu" name="menuDetection">
    <property name="title">
     <string>Detection</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="actionAutomatic"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuMirror">
     <property name="title">
      <string>Mirror</string>
     </property>
     <widget class="QMenu" name="menuImage">
      <property name="title">
       <string>Image</string>
      </property>
      <addaction name="actionMirrorImageHorizontally"/>
      <addaction name="actionMirrorImageVertically"/>
     </widget>
     <widget class="QMenu" name="menuScrews">
      <property name="title">
       <string>Screws</string>
      </property>
      <addaction name="actionMirrorScrewsVertically"/>
      <addaction name="actionMirrorScrewsHorizontally"/>
      <addaction name="separator"/>
      <addaction name="actionMirrorTightLose"/>
     </widget>
     <addaction name="menuImage"/>
     <addaction name="menuScrews"/>
    </widget>
    <addaction name="actionViewBounding_Box"/>
    <addaction name="actionCalculated_Center"/>
    <addaction name="separator"/>
    <addaction name="actionViewScrews"/>
    <addaction name="separator"/>
    <addaction name="menuMirror"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDetection"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImage_processing_settings">
   <property name="text">
    <string>Image processing...</string>
   </property>
  </action>
  <action name="actionSave_screenshot">
   <property name="text">
    <string>Save screenshot</string>
   </property>
  </action>
  <action name="actionClose_2">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionAutomatic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatic</string>
   </property>
  </action>
  <action name="actionViewBounding_Box">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bounding Box</string>
   </property>
  </action>
  <action name="actionCalculated_Center">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Calculated Center</string>
   </property>
  </action>
  <action name="actionViewScrews">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Screws</string>
   </property>
  </action>
  <action name="actionSettingsScrews">
   <property name="text">
    <string>Screws...</string>
   </property>
  </action>
  <action name="actionDebug_Image">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Debug Image</string>
   </property>
  </action>
  <action name="actionConnection">
   <property name="text">
    <string>Connection...</string>
   </property>
  </action>
  <action name="actionMirrorImageHorizontally">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Horizontally</string>
   </property>
  </action>
  <action name="actionMirrorImageVertically">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Vertically</string>
   </property>
  </action>
  <action name="actionMirrorScrewsVertically">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Vertically</string>
   </property>
  </action>
  <action name="actionMirrorScrewsHorizontally">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Horizontally</string>
   </property>
  </action>
  <action name="actionMirrorTightLose">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tight/Lose</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
